### ISSUE TITLE
Broken link to dpv document

### ISSUE THREAD

**Issue by tanjadegroot** [camaraproject/IdentityAndConsentManagement#203]
**Problem description**
in the https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md, section 11.6, the link to the "purpose section" is broken. 

**Expected behavior**
Rewrite the section to remove the link to the external doc from this doc (which can just point to the ICM profile doc) to avoid maintenance problems between these 2 documents. Only keep the external link to W3C in the ICM doc.

**Alternative solution**
Fix the broken link to point to v2.0 (but I would avoid) 

**Additional context**

**Comment by jpengar** [comment]
@tanjadegroot This same issue was raised in https://github.com/camaraproject/IdentityAndConsentManagement/issues/195 for ICM documentation, fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/196, and resulted in #202 to generate a r0.2.1 patch release with the fix. But if you're referring to https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md, shouldn't this issue be opened in Commonalities WG, and perhaps refer to ICM's #195 for further context?

**Comment by tanjadegroot** [comment]
Oops, sorry indeed, I picket the wrong project :-( Is there a way to move the issue ?

**Comment by jpengar** [comment]
There is a `Transfer issue' option, but I'm not allowed to transfer this issue to the Commonalities subproject. Not sure if @hdamker or others can do this. If not, we can close this issue and just open a copy of it in Commonalities.

**Comment by hdamker** [comment]
> @hdamker or others can do this. If not, we can close this issue and just open a copy of it in Commonalities.

Yes, I can do (requirement is write access in both repositories).
